src/shared/canvas/MyData/MyData.tsx

```diff
@@ -71,14 +97,27 @@ const HomeMyData: React.FC<{}> = () => {
   };
 
   const { data: resources, isLoading } = useQuery({
-    queryKey: ['my-data-resources', { size, offset, query }],
+    queryKey: [
+      'my-data-resources',
+      { size, offset, query, locate, issuer, sort: JSON.stringify(sort) },
+    ],
     queryFn: () =>
       nexus.Resource.list(undefined, undefined, {
         size,
         from: offset,
-        // after: offset,
-        q: query,
+        [issuer]: issuerUri,
+        ...(locate && query.trim().length
+          ? {
+              locate: query,
```

Comment:
I found this piece a bit difficult to understand. What is `locate` here? 
Could we also document here somehow that the use of `issuerUri` will not be needed when the changes are made on delta side :pray:

Reply:
`locate` is the same value that we pass to the API endpoint (it's for the checkbox when the user want to filter by the `@id` of `_self`)

for the issuerUri, I think it's self explanatory of what is it specificly in the types `issuerUri: 'createdBy' | 'updatedBy'`
and the API documentation is pretty clear also, of how to use this two fields.